### ISSUE TITLE
cli.ini boolean options set when False

### ISSUE THREAD

**Issue by corburn** [certbot/certbot#4407]
## My operating system is (include version):
linux alpine edge

## I installed Certbot with (certbot-auto, OS package manager, pip, etc):
pip install certbot==0.12.0

## I ran this command and it produced this output:

I created `/etc/letsencrypt/cli.ini` listing all options, even if they matched the default:

```
# /etc/letsencrypt/cli.ini

authenticator = webroot
webroot-path = /srv
dry-run = False
# ...
```

Command:

```
$ certbot certonly
```

Output:
```
<snip>
IMPORTANT NOTES:
 - The dry run was successful
```

## Certbot's behavior differed from what I expected because:

I expect setting a boolean option to `True` would be the same as setting the flag:
```
dry-run = True
```

```
certbot certonly --dry-run -d example.com
```

I expect setting a boolean option to `False` would be the same as omitting the flag:

```
dry-run = False
```

```
certbot certonly -d example.com
```

The boolean flags were set regardless of whether they were True or False. The option just needs to be present in the configuration file.

**Comment by SwartzCr** [comment]
@bmw this doesn't seem like intended behavior - is this a problem with our config parsing wrt argparse?

**Comment by bmw** [comment]
This is an issue with `configargparse` which is the library we use to parse configuration files. I filed an issue at bw2/configargparse#87.

**Comment by bmw** [comment]
I believe this was fixed in configargparse 0.12.0. We should update the pinned version of configargparse in letsencrypt-auto so users benefit.

**Comment by SwartzCr** [comment]
any update on pinning?
